homeassistant/components/ecowitt/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "config": {
+    "error": {
+      "invalid_port": "Port is already used.",
+      "unknown": "Unknown error."
+    },
+    "step": {
+      "user": {
+        "description": "The following steps must be performed to set up this integration.\n\nUse the Ecowitt App (on your phone) for your Ecowitt WebUI at the device.\nPick your station -> Menu Others -> DIY Upload Servers.\nHit next and select 'Customized'\n\nPick the protocol Ecowitt, and put in the ip/hostname of your hass server.\nPath have to match, you can copy with secure token /.\nSave configuration. The Ecowitt should then start attempting to send data to your server.",
```

Comment:
What do we mean with this?

> Use the Ecowitt App (on your phone) for your Ecowitt WebUI at the device.

Is the WebUI a special menu inside the app? Why do we say "at the device"?

Reply:
I updated the text. That is from one of the custom components